dd-sdk-android/src/main/kotlin/com/datadog/android/Datadog.kt

```diff
@@ -284,6 +304,11 @@ object Datadog {
         "If you need it, submit an issue at https://github.com/DataDog/dd-sdk-android/issues/"
 
     internal const val SHUTDOWN_THREAD = "datadog_shutdown"
+    internal const val ENV_NAME_VALIDATION_REG_EX = "[a-zA-Z0-9_:./-]{0,195}[a-zA-Z0-9_./-]"
```

Comment:
As it must also [start with a letter](https://docs.datadoghq.com/getting_started/tagging/#defining-tags) I think it should be:
```
[a-zA-Z][a-zA-Z0-9_:./-]{0,194}[a-zA-Z0-9_./-]
```

Reply:
it's a confusion because the tag name will be : `env:something` so it will always start with a name we don't have to check for that. Also from here the max length of 196 characters as the rest of 4 is taken by `env:`

Reply:
All clear 👌